### DISCUSSION TITLE
SAE Webseite nicht erreichbar

### DISCUSSION THREAD

**Discussion by HerwigHahn** [camueller/SmartApplianceEnabler#556] [Installation und Betrieb]
Auf dem Handy habe ich (und  meine Frau) den SAE-Staus als Chrome Lesezeichen abgespeichert. Die Anzeige hat auch mehrere Wochen gut funktioniert. Nun wird bei uns beiden  angezeigt: "Webseite nicht erreichbar". Auf dem PC ist die gleiche Website weiterhin erreich- und nutzbar. 
Hat jemand einen Tipp, warum das so ist und wie ich den Status wieder auf dem Handy angezeigt bekommen kann?

**Comment by camueller** [comment]
Die SAE-Weboberfläche ist ja nur im lokalen Netz verfügbar, während das Handy ja auch das Mobilfunknetz nutzen kann, wenn das WLAN ausgeschaltet ist. Wenn das Handy das WLAN nutzt, in dem sich auch der SAE befindet, sollte die SAE-Weboberfläche funktionieren.

**Reply by HerwigHahn** [reply]
Nach (erneutem) Neustart des Raspberry ist es jetzt so, dass auf den neuen Smartphone (mit aktuellem Android update vom 29.09.23) weiterhin die Fehlemessage lautet: Webseite nicht erreichbar, dns_probe_finished_nxdomain error. Beim Tablet mit Anroid 8.1 fuktioniert es. 
Habe alles ausprobiert was es an Einstellungsänderungen zu dem Problem unter Android zu finden gab. Ledier ohne Erfolg. 
 Das nur zur so zur Info (Ist je kein Problem von SAE), falls jemand das gleiche Problem hat.  
Und vielleicht hat ja auch jemand eine Lösung.

**Comment by HerwigHahn** [comment]
Danke für die Info. 

Ist aber so, dass beide Handys in dem lokalen WLAN sind. 

Und es ging ja, ohne dass Einstellungen geändert wurden, vorher schon mal.

Habe sowohl Moden wie auch mein Handy neu gestartet.
